libs/vscode/webview/src/lib/get-task-execution-schema.ts

```diff
@@ -94,24 +93,24 @@ export async function getTaskExecutionSchema(
             return;
           }
 
-          const builderOptions = await readBuilderSchema(
-            cliTaskProvider.getWorkspacePath(),
-            selection.architectDef.builder
+          const { validBuilder, options } = await verifyBuilderDefinition(
+            selection.projectName,
+            selection.command,
+            json
           );
-
-          if (!builderOptions) {
+          if (!validBuilder) {
             return;
           }
 
           return {
             ...readArchitectDef(
```

Comment:
Gets configurations: Arguments weren't quite right here. It was passing for example `readArchitectDef('run', 'build', 'store')` but it needs the architectDef from the selected target to get the configurations.